tests/testthat/test-int-latent_individual.R

```diff
@@ -85,6 +85,43 @@ test_that("epidist.epidist_latent_individual Stan code has no syntax errors and
   expect_no_error(mod_gamma$compile())
 })
 
+test_that("epidist.epidist_latent_individual fits and the MCMC converges in the gamma delay case", { # nolint: line_length_linter.
+  # Note: this test is stochastic. See note at the top of this script
+  skip_on_cran()
+  set.seed(1)
+  fit_gamma <- epidist(
+    data = prep_obs_gamma,
+    family = stats::Gamma(link = "log"),
```

Comment:
Here and elsewhere I think we should be silencing progress messages during testing
 Not doing so makes the output hard to pass as it is so noisy.

Reply:
Do you have suggestions about how to do that? I'm looking in `epinowcast` now